log_project.go

```diff
@@ -452,3 +452,91 @@ func (p *LogProject) RemoveConfigFromMachineGroup(confName, groupName string) er
 
 	return err
 }
+
+// CreateETLJob creates a new ETL job in SLS.
+func (p *LogProject) CreateETLJob(j *ETLJob) error {
+	body, err := json.Marshal(j)
+	if err != nil {
+		return NewClientError(err.Error())
+	}
+
+	h := map[string]string{
+		"x-log-bodyrawsize": fmt.Sprintf("%v", len(body)),
+		"Content-Type":      "application/json",
+		"Accept-Encoding":   "deflate", // TODO: support lz4
+	}
+	_, _, err = request(p, "POST", "/etljobs", h, body)
+	return err
+}
+
+// GetETLJob returns ETL job according to job name.
+func (p *LogProject) GetETLJob(name string) (*ETLJob, error) {
+	h := map[string]string{
+		"x-log-bodyrawsize": "0",
+	}
+
+	_, buf, err := request(p, "GET", "/etljobs/"+name, h, nil)
+	if err != nil {
+		return nil, err
+	}
+	s := &ETLJob{}
```

Comment:
Looks like the ETLJob in the get response has more fields. Also, name it etlJob or job?